event-istrusted/index.html

```diff
@@ -0,0 +1,52 @@
+---
+feature_name: Event.isTrusted
+chrome_version: 46
+feature_id: 6461137440735232
+---
+
+<style>
+  button {
+    display: block;
+    margin: 10px auto;
+  }
+  #keanu {
+    height: 50px;
+    border-radius: 50%;
+    border: 2px solid;
+    display: block;
+    margin: 0 auto;
+    cursor: pointer;
+  }
+</style>
+
+<p>The <code>isTrusted</code> read-only property of the <code>Event</code>
+interface is a <code>Boolean</code> that is <code>true</code> when the event was generated
+by a user action such as mouse click for instance, and <code>false</code> when
+the event was created or modified by a script or dispatched via
+<code>dispatchEvent</code>.</p> <p>This new property is intended primarily for
+use by browser extensions, to determine if an event was dispatched by a script running
+in the main world or not.</p>
+
+<img id="keanu" src="Conspiracy-Keanu.jpg" title="Click me...">
+<button>BUTTON</button>
```

Comment:
Something like "Generate Trusted Event" would be a more descriptive title than "BUTTON".